tests/spread/general/multipass/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: Build a basic snap using multipass and ensure that it runs
+warn-timeout: 9m  # Keep less than 10 minutes so Travis can't timeout
+priority: 90  # Run this test relatively early since fetching images can take time
+
+# Keep this 18.04 only for now as it is the only stable base.
+systems: [ubuntu-18.04-64]
+
+environment:
+  SNAP_DIR: snaps/make-hello
+
+prepare: |
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "$SNAP_DIR/snap/snapcraft.yaml"
+
+restore: |
+  cd "$SNAP_DIR"
+  snapcraft clean
+  rm -f ./*.snap
+
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  restore_yaml "snap/snapcraft.yaml"
+
+execute: |
+  cd "$SNAP_DIR"
+
+  # Ensure that snapcraft uses build VMs
+  unset SNAPCRAFT_BUILD_ENVIRONMENT
```

Comment:
this could have been in `environment`, right? But yes, I would have started the same way :wink: